src/components/Support/index.tsx

```diff
@@ -47,7 +47,6 @@ const SupportPage: React.FC = () => (
                       src="https://discordapp.com/widget?id=485586884165107732&theme=light"
                       width="350"
                       height="500"
-                      allowTransparency
```

Comment:
This prop caused the lint check to fail because this prop doesn't actually do anything on an `iframe` in React. If you look at the result of this component on dvc.org, there's no allowTransparency attribute in sight.